llvm/test/Transforms/LoopVectorize/iv-select-cmp-decreasing.ll

```diff
@@ -941,6 +941,205 @@ exit:                                             ; preds = %loop
   ret i16 %spec.select.lcssa
 }
 
+; The signed sentinel value for decreasing-IV vectorization is LONG_MAX, and since
+; the IV hits this value with smin vectorization, it needs to be vectorized with a
+; an unsigned sentinel and umin instead.
+define i64 @select_decreasing_induction_icmp_iv_unsigned(ptr %a) {
```

Comment:
Just double-checking, do we have a negative test case where we hit unsigned max?

Reply:
Oops, I forgot. Thanks for the reminder.